client/src/pages/organizations/Show.js

```diff
@@ -270,6 +271,33 @@ const OrganizationShow = ({ pageDispatchers }) => {
     reportQueryParams.orgRecurseStrategy = RECURSE_STRATEGY.CHILDREN
   }
 
+  function showLeadingPositions(positions, role, label) {
+    return (
+      positions &&
+      positions.some(pos => pos.role === role) && (
+        <Field
+          name={label}
+          component={FieldHelper.ReadonlyField}
+          label={label}
+          humanValue={
+            <ListGroup>
+              {positions.map(
+                pos =>
+                  pos.role === role && (
+                    <ListGroupItem key={pos.uuid}>
+                      <LinkTo modelType="Position" model={pos}>
+                        {pos.name}
+                      </LinkTo>
+                    </ListGroupItem>
+                  )
+              )}
+            </ListGroup>
+          }
+        />
+      )
+    )
+  }
```

Comment:
Please extract this function (almost the same as OrganizationPreview) into a util function. And `<LinkTo>` doesn't need children.
You're also looping twice over the positions list, which is unnecessary. You could extract a function like:
```javascript
function getPositionsForRole(positions, role) {
  const positionList = positions
    .filter(pos => pos.role === role)
    .map(pos => (
      <ListGroupItem key={pos.uuid}>
        <LinkTo modelType="Position" model={pos} />
      </ListGroupItem>
    ))
  if (!_isEmpty(positionList)) {
    return <ListGroup>{positionList}</ListGroup>
  }
}
```
and call it as:
```javascript
  function showLeadingPositions(positions, role, label) {
    const positionList = getPositionsForRole(positions, role)
    if (!_isEmpty(positionList)) {
      return (
        <Field
          name={label}
          component={FieldHelper.ReadonlyField}
          label={label}
          humanValue={positionList}
        />
      )
    }
  }
```

[And in OrganizationPreview, you would call it as:
```javascript
  function showLeadingPositions(positions, role, label) {
    const positionList = getPositionsForRole(positions, role)
    if (!_isEmpty(positionList)) {
      return <PreviewField label={label} value={positionList} />
    }
  }
```
]

And please also move the `showLeadingPositions()` function declaration to the end of the component (i.e. after the main return).